server_environment/__manifest__.py

```diff
@@ -21,7 +21,7 @@
 
 {
     "name": "server configuration environment files",
-    "version": "9.0.1.2.0",
+    "version": "10.0.1.2.0",
```

Comment:
@sebalix Is it the best pratice to reset complety the version ?
I find it a little strange. it's necessary to change the version of Odoo but the version of the module remains the same.